### PR TITLE
Removed deprecated package shortid

### LINKED ISSUE #1899: Replace deprecated package `shortid`
**Describe the bug**

1. shortid is no longer supported

**To Reproduce**

- Run `npm install`. You'll see this messag

```
npm WARN deprecated shortid@2.2.16: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.

```

**Expected behavior**

1. No error because a currently maintained equivalent package is used
2. All functionality is maintained
3. All tests are valid and code coverage is maintained

**Actual behavior**

- See above

**Screenshots**

- N/A

**Additional details**

- N/A

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by Sahi1l-Kumar**
**What kind of change does this PR introduce?**
A bugfix. It removes the deprecated package shortid

**Issue Number:**
Fixes #1899 

**Did you add tests for your changes?**
No.

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

**Summary**
The following pull request removes the deprecated package `shortid` which is no longer supported.

**Does this PR introduce a breaking change?**
No.

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**
Yes.

### CODE REVIEW

File: src/utilities/uploadImage.ts [hunk 1]
```diff
@@ -33,6 +34,7 @@ export const uploadImage = async (
         ),
       )
       .on("close", resolve)
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (palisadoes) [inline src/utilities/uploadImage.ts:37]: Why was this added? How can you rescue the code to eliminate the explicit any?

**Review by palisadoes — CHANGES_REQUESTED** [review]

**Comment by Sahi1l-Kumar** [comment]
Added this comment `eslint-disable-next-line @typescript-eslint/no-explicit-any` in an old file which was showing error because of
`any` type.
Please review this pull request . Thank you.

File: src/utilities/uploadImage.ts (hunk 1 above)
Comment (Sahi1l-Kumar) [reply src/utilities/uploadImage.ts:37]: I wasn't able to commit without removing this error locally.

Comment (Sahi1l-Kumar) [reply src/utilities/uploadImage.ts]: I have added they type to paramater `error` now

**Comment by Sahi1l-Kumar** [comment]
@palisadoes  I have addressed the requested changes.

**Review by palisadoes — APPROVED** [review]

**Pull request merged**